pages/common/pg_restore.md

```diff
@@ -0,0 +1,23 @@
+# pg_restore
+
+> Restore a PostgreSQL database from an archive file created by pg_dump
+
+- restore an archive into an existed database
+
+`pg_restore -d {{db_name}} {{archive_file.dump}}`
+
+- customize host, port, username
+
+`pg_restore -h {{localhost}} -p {{5432}} -U {{username}} -d {{db_name}} {{archive_file.dump}}`
```

Comment:
localhost it is a alias for concrete address.
In the example it should be one of two:

```md
`pg_restore -h {{host}}`
`pg_restore localhost`
```